### ISSUE TITLE
Add a translatable key under a11y in JSON files

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/rosetta#449]
From #132 -

> we think a new key in the JSON (perhaps like "a11y_rosetta") could be used to section accessibility strings that should be translatable through Rosetta. Rosetta could display those for translation, possibly with some additional information about what "accessibility strings" are.

> we are going to continue with https://github.com/phetsims/rosetta/issues/132#issuecomment-2652339870 using a key called translatable, which will be under the a11y key in the JSON file.

One issue with this idea is that it will create duplication for accessibility strings that are not translatable. Especially to have consistent nesting.

```json
{
  "a11y": {
    "translatable": {
      "snapshotButton": {
        "accessibleName": {
          "value": "Take Snapshot"
        },
        "accessibleHelpText": {
          "value": "Take up to {{max}} snapshots of spectrometer data."
        }
      },
    }
  },
  "snapshotButton": {
      "contextResponse": {
         "value": "The snapshot has {{count}} {{color}} {{particleType}}."
      }
   }
}
```

Alternatively, we could use metadata like `"translatable: false"` on each value.

**Comment by jessegreenberg** [comment]
However we finalize things here, we need to make sure that babel is capturing the translatable strings. Running `grunt modulify` should add entries to `_generated_development_strings_`.

**Comment by jessegreenberg** [comment]
After checking in during dev meeting, we decided not to use a `translatable` key, and to add some metadata for this instead. Probably `translatable: false`.

- This reduces duplication described in https://github.com/phetsims/rosetta/issues/449#issue-2864506066. It keeps related strings closer together (so don't have to put related strings in and out of translatable key)
- The intent is a better fit for metadata.

To roll this out, we will 
- Put `translatable: false` on all existing accessibility strings (unless they are currently under the `translatable` key)
- Remove the `translatable` key (trig-tour, quantum-measurement, MOTHA, and common code so far).
- Among rosetta's other work, we need to add support for `translatable: false`.

**Comment by jessegreenberg** [comment]
The first thing we need to do now is revert the addition of the `translatable` key in strings json files.

**Comment by jessegreenberg** [comment]
That is done. Since this issue was about adding the key, we should close it. Next steps were moved to the original issue, https://github.com/phetsims/rosetta/issues/132.
